src/system/BlocksModule/AbstractBlockHandler.php

```diff
@@ -103,7 +103,7 @@ public function getType()
     public function setContainer(ContainerInterface $container = null)
     {
         $this->container = $container;
-        $this->setTranslator($container->get('translator')); // for TranslatorTrait
+        $this->setTranslator($container->get('translator.default')); // for TranslatorTrait
```

Comment:
@Guite this _may_ have been the reason some translating wasn't working as expected.